config.example.yaml

```diff
@@ -119,7 +120,22 @@ templates:
                   get:
                     backend_request:
                       uri: http://graphoid.wikimedia.org/{domain}/v1/png/{title}/{revision}/{graph_id}
-                      body: '{$.request.body}'
+
+      /{module:mathoid}:
+        x-modules:
+          - name: simple_service
+            version: 1.0.0
+            type: file
+            options:
+              paths:
+                /v1/{format}/{formula}/{type}:
+                  get:
+                    backend_request:
+                      method: post
+                      uri: http://mathoid-tester.wmflabs.org/{format}
```

Comment:
I'd vote for http://mathoid.testme.wmflabs.org/
Note the port here is 80 and not 10042